app/controllers/users_controller.rb

```diff
@@ -21,6 +21,10 @@ def team_members
     end
   end
 
+  def team_leader
```

Comment:
Is this to identify the Team Leader of the person logged in? If so, how would it differ from `current_user.team_leader` (which is already implemented in the model)?

Reply:
deleted